tests/plugins/filebrowser/uploadbutton.js

```diff
@@ -192,8 +191,7 @@
 				// Execute just after XHR request is generated;
 				editor.once( 'fileUploadRequest', function() {
 					resume( function() {
-						assert.areSame( 1, this.requests[ 0 ].url.match( /responseType=json/g ).length,
-							'responseType parameter count' );
+						assert.isMatching( /responseType=json/g, this.requests[ 0 ].url, 'responseType parameter was added' );
```

Comment:
This change defeats the purpose of this test as it does not longer check if parameter was added twice.

Reply:
Make sense, pushed the change.